docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,15 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is a non-scoring <<glossary-query,query>>, meaning that it does not score documents.
+  It is only concerned about answering the question - "Does this document match?". 
+  The answer is always a simple, binary yes or no. This kind of query is said to be made 
+  in a <<https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html,"filtering" context>>, 
```

Comment:
Unfortunately this way of adding the link doesn't work, when I build the page I get a link validity error. You should use a link Id like in the other cases in this paragraph. In this case `<<query-filter-context,filter context>>` should work. I would also leave out the quotes and change the link text to "filter context" since that is the what it is called in the page that is refered to.

Reply:
Check it and  see now, but it is not looking good on my end

Reply:
What do you mean by "not looking good on my end"? The current update still contains the absolute link and that will break our docs build, so this will need changing unfortunately. What do you run currently to check if the docs are built correctly?

Reply:
I think it is okay now, I didn't pay attention to what I doing wrong until you mentioned about the "absolute link" again. Hope is fixed now.